cmake/kokkos_arch.cmake

```diff
@@ -221,6 +221,7 @@ ENDIF()
 IF(KOKKOS_ARCH_NATIVE)
   COMPILER_SPECIFIC_FLAGS(
     COMPILER_ID KOKKOS_CXX_HOST_COMPILER_ID
+    MSVC    NO-VALUE-SPECIFIED
```

Comment:
Why is that the right thing to do rather than error out and say the option is not available?
The proposed resolution is likely so surprise users.